src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__wrap calendar__wrap--month-length--31">
+        <div class="calendar__day calendar__day--start-day--mon"></div>
```

Comment:
each modifier should be implemented for `calendar` block - not for the first day element or wrappers